redfish-core/lib/account_service.hpp

```diff
@@ -2197,6 +2201,15 @@ inline void requestAccountServiceRoutes(App& app)
                                 "Location",
                                 "/redfish/v1/AccountService/Accounts/" +
                                     username);
+#ifdef BMCWEB_ENABLE_LINUX_AUDIT_EVENTS
```

Comment:
Can we do this in a generic way? I really really don't think we want these all over the code for maintainability concerns

Reply:
Sorry, I should have been more clear in my comment, I think we should do this in like connection.hpp, request.hpp, or response.hpp in a generic way